src/pages/suzin/Login/LoginSuzin.js

```diff
@@ -1,7 +1,58 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+// import { Link } from "react-router-dom";
+import { useNavigate } from 'react-router-dom';
+import './LoginSuzin.scss';
 
-function LoginSuzin() {
-  return <h1>hi</h1>;
-}
+const LoginSuzin = () => {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [bt, setBt] = useState('skyblue');
+  const navigate = useNavigate();
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+  const saveUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  useEffect(() => {
+    id.includes('@') && pw.length > 4 ? setBt('blue') : setBt('skyblue');
+  }, [id, pw]);
+
+  return (
+    <div id="loginking">
```

Comment:
```suggestion
    <div className="loginSuzin">
```
가장 부모 태그의 className은 컴포넌트의 이름으로 해주세요!